pages/events/[type].js

```diff
@@ -49,18 +54,20 @@ export default function EventsPage() {
   }
   return (
     <>
-      <Link href="/">Home</Link>
-      <h1>
+      <StyledHeadline>
         Your {type} {type === "wishlist" ? null : "Appointments"}
-      </h1>
-      <button
+      </StyledHeadline>
+
+      <StyledAddButton
         onClick={() => {
           setIsEditing(!isEditing);
         }}
       >
-        Add Appointment
-      </button>
-      {isEditing ? <AddEventForm onSubmit={handleSubmit} /> : null}
+        <SVGIcon variant={isEditing ? "cancel" : "add"} width="40px" />
+      </StyledAddButton>
+      {isEditing ? (
+        <AddEventForm onSubmit={handleSubmit} appointment={{ type: type }} />
+      ) : null}
```

Comment:
this would be even easier to read with a logical and:

```suggestion
      {isEditing && (
        <AddEventForm onSubmit={handleSubmit} appointment={{ type: type }} />
      )}
```

Reply:
Doesn't use "&&" often, looks nice